staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -727,6 +747,13 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 			// construct FrontendIPConfigurationPropertiesFormat
 			var fipConfigurationProperties *network.FrontendIPConfigurationPropertiesFormat
 			if isInternal {
+				// azure does not support ILB for IPv6 yet.
+				// TODO: remove this check when ILB supports IPv6 *and* the SDK
+				// have been rev'ed to 2019* version
+				if utilnet.IsIPv6String(service.Spec.ClusterIP) {
```

Comment:
This change isn't feature gated, is that okay because we already assume ClusterIP can never by v6 for Azure internal LB?

Reply:
This change is not gated, because we want to be able to run IPv6 only clusters on Azure. Also, ILB on Azure does not support IPv6 on ILB (it is in the release notes of this PR)